jujugui/static/gui/src/app/app.js

```diff
@@ -727,6 +729,9 @@ YUI.add('juju-gui', function(Y) {
       @method _renderUserProfile
     */
     _renderUserProfile: function() {
+      var fetchMacaroonCallback = function(error, macaroon) {
```

Comment:
I'm not sure why this wrapper is necessary, why not pass the `_storeUser` method directly?